encrypted-media-respec.html

```diff
@@ -3942,7 +3941,7 @@
               <li><p>MUST be sent to a URL selected independently of any origin and application.</p></li>
             </ul>
             <p>
-              Implementations MUST NOT expose, even in encrypted form, <a def-id="origin">origin(s)</a>-, origin- or application-specific information, or values that are <a def-id="associable"></a> with origin(s) to centralized servers since this could create a central record of all origins visited by a user or device.
+              Implementations MUST NOT expose, even in encrypted form, <a def-id="origin">origin(s)</a>, origin- or application-specific information, or values that are <a def-id="associable"></a> with origin(s) to centralized servers since this could create a central record of all origins visited by a user or device.
```

Comment:
Fixes issue identified in https://github.com/w3c/encrypted-media/issues/272#issuecomment-234428862.